assembly/driver/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>pentaho</groupId>
+    <artifactId>pdi-dataservice-client-assemblies</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>pdi-dataservice-driver-bundle</artifactId>
+  <version>8.1.0.0-SNAPSHOT</version>
```

Comment:
This version is not needed. You can inherit it from the parent.

Reply:
Legacy code :) It was like that on the previous driver assembly pom... will fix, good catch ;)